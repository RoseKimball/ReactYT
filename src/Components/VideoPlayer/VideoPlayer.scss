.video-container {
    width: 100%;
    position: relative;
    // padding-top: ;
    // display: block;
    // content: "";

}

.video-player {
    min-height: 480px;
    width: 59.25%;
    height: 100%;
    position: absolute;
    top: 0;

}