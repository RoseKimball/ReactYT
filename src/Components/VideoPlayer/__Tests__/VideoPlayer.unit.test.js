import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, mount, render } from 'enzyme';
import VideoPlayer from '../VideoPlayer';

describe('VidePlayer', () => {
    test('renders VideoPlayer', () => {
        const wrapper = shallow (
            <VideoPlayer id='GPJDXcGYkqU'/>
        )
        expect(wrapper).toMatchSnapshot();
    })
    test('renders null without id prop', () => {
        const wrapper = shallow (
            <VideoPlayer/>
        )
        expect(wrapper).toMatchSnapshot();
    })
})