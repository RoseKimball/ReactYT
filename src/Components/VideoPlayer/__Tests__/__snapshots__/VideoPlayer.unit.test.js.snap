// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VidePlayer renders VideoPlayer 1`] = `ShallowWrapper {}`;

exports[`VidePlayer renders null without id prop 1`] = `ShallowWrapper {}`;
